inference-engine/src/transformations/src/transformations/softplus_decomposition.cpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/softplus_decomposition.hpp"
+
+#include <memory>
+#include <vector>
+
+#include <ngraph/opsets/opset4.hpp>
+#include <ngraph/rt_info.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+
+ngraph::pass::SoftPlusDecomposition::SoftPlusDecomposition() {
```

Comment:
Need to implement SoftPlus fusing transformation and included it to the CommonOptimizations passes

Reply:
Done.